capture.py

```diff
@@ -831,6 +831,7 @@ def _maintain_sequence_time_panel():
     # If a panel is set to sequence time, it will grab focus from the 
     # independent panel during the playblast. There's no way to unset it, so we
     # create a dummy panel, set it to sequence time, then delete it.
+    cmds.sequenceManager(query=True, writableSequencer=True)
```

Comment:
Do include a note as to why this is here, otherwise it might be removed by someone in the future looking at this thinking it's unused/unnecessary.

Reply:
> Do include a note as to why this is here, otherwise it might be removed by someone in the future looking at this thinking it's unused/unnecessary.

Done!